syntax = "proto3";

package example.complex;

message Dummy {
    uint32 id = 1;
    string name = 2;
}

message Complex {
    Dummy single_dummy = 1;
    repeated Dummy list_of_dummy = 2;
}